### ISSUE TITLE
[6.x] Gate of the Setting Sun

### ISSUE THREAD

**Issue by LuigiElleBalotta** [TrinityCore/TrinityCore#17018]
**Description**:
This issue is opened to help this guys testing his scripts: https://github.com/TrinityCore/TrinityCore/pull/17011
**Current behaviour**: Tell us what happens
Gate of the Setting Sun is not spawned at all
**Expected behaviour**: Tell us what should happen instead
Gate of the Setting Sun scripted
**Steps to reproduce the problem**:

1. Enter the dungeon
2. Opppsss, it is empty
3. 

**Branch(es)**: 335/6x  
6.x
**TC hash/commit**:  https://github.com/TrinityCore/TrinityCore/commit/114353f94b6cd75d233c9c1cb6fdd207755cf9f5

**TDB version**:  6.04

**Operating system**:  Win 10

**Linking model**: static/dynamic
Both

[//]: # (This template is for problem reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

**Comment by msoky** [comment]
you can get in your pull request https://github.com/TrinityCore/TrinityCore/pull/17011

**Comment by LuigiElleBalotta** [comment]
@msoky it is not my pr

**Comment by msoky** [comment]
oh right..sorry, i read only description

**Comment by MitchesD** [comment]
it's just raw taken from WPP? if so, then we can't accept it, you have to check for double spawns, phasing etc.

**Comment by AMDIvailo** [comment]
I tested it myself. No duplicates, everything looks fine. There are some missing floors at the ground, which i think are gameobjects. There are a lot of missing gameobjects, that maybe must be spawned from script.

**Comment by LuigiElleBalotta** [comment]
Yes, as I said, there are some events (explosion) that changes the area objects, they must be done by script to do it offylike
